website/content/docs/concepts/architecture.mdx

```diff
@@ -162,14 +176,129 @@ on hardware features such as architecture and availability of GPUs, or software
 like operating system and kernel version, or they can be business constraints like
 ensuring PCI compliant workloads run on appropriate servers.
 
+# Client Organization
+
+When the Nomad [scheduler][] receives a job registration request, it needs to
+determine which clients will run allocations for the job. By default, all nodes
+are considered for placements but this process can be adjusted by cluster
+operators and job submitters to achieve more control over where allocations
+are placed.
+
+The steps to achieve this control consists of setting certain values in
```

Comment:
If this paragraph is used to introduce Affinities and Constraints, Datacenter, Nodepools... it would be worth mentioning them.

Reply:
Hum...I don't quite follow. I think the confusion may be that I'm using "setting" as the gerund of the verb "to set", not as "configuration".

But this paragraph is gone once I moved this section to the new Scheduling -> Placements page 😅